openshift/templates/aries-mediator-agent/aries-mediator-agent-deploy.askar.prod.param

```diff
@@ -0,0 +1,45 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/aries-mediator-agent/aries-mediator-agent-deploy.yaml
+#=========================================================
+# NAME=aries-mediator-agent
+# APP_GROUP=aries-mediator-service
+# NAMESPACE_NAME=4a9599
+# APP_NAME=aries-mediator-service
+SUFFIX=-askar
+# ROLE=agent
+# SERVICE_ROLE=cotroller
+# IMAGE_NAMESPACE=4a9599-tools
+# AGENT_READ_ONLY_LEDGER=false
+# ACAPY_WS_HEARTBEAT_INTERVAL=15
+# ACAPY_WS_TIMEOUT_INTERVAL=60
+# ADMIN_API_KEY=[a-zA-Z0-9]{32}
+TAG_NAME=prod
+# WALLET_HOST=aries-mediator-db
+# WALLET_PORT=5432
+# CONFIG_MOUNT_PATH=/home/indy/configs/
+# CONFIG_FILE_NAME=mediator-auto-accept.yml
+# CONFIG_MAP_NAME=mediator-config
+# MEDIATOR_ARG_FILE=./configs/mediator-auto-accept.yml
+ACAPY_WALLET_TYPE=indy
```

Comment:
Is this line right?

Reply:
Ah…nevermind — I thought this was a change in aries-mediator-service, not the deployment repo.